src/Options.js

```diff
@@ -60,6 +61,7 @@ class Options {
     textSize: number;
     phantom: boolean;
     font: string | void;
+    fontStyles: Array<string>;
```

Comment:
I think changing `font` to `baseFont` or `fontFamily` would make it clearer that the two are necessary and work together.

Reply:
According to the answer to https://tex.stackexchange.com/questions/5008/when-to-use-bold-italics-small-caps-typewriter-etc there are three axes for fonts:
- The shape axis covers: \textup, \textit, \textsc and \textsl
- The family axis covers: \textrm, \textsf, \texttt
- The weight axis covers: \textbf, \textmd

Maybe we should change from `font` + `fontStyles` array to three props:
- `fontShape`
- `fontFamily`
- `fontWeight`